ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/byo-kms/etcd-encryption/provision/cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption-provision-chain.yaml

```diff
@@ -0,0 +1,25 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption-provision
+  steps:
+  - ref: aws-provision-vpc-shared
```

Comment:
In this case, do we have to combine customer vpc?

Reply:
ahh, you're right, will remove it.